src/components/WalletConnect/SignMessageRequest.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {
+  StyleSheet,
+  View,
+  ScrollView,
+  TouchableWithoutFeedback,
+  Text,
+} from 'react-native';
+import { useDispatch } from 'react-redux';
+import { useNavigation } from '@react-navigation/native';
+import { t } from 'ttag';
+import {
+  walletConnectAccept,
+  walletConnectReject
+} from '../../actions';
+import { COLORS } from '../../styles/themes';
+import NewHathorButton from '../NewHathorButton';
+import { DappContainer } from './NanoContract/DappContainer';
+import { commonStyles } from './theme';
+import { NanoContractIcon } from '../Icons/NanoContract.icon';
+
+export const SignMessageRequestData = ({ data }) => (
```

Comment:
Do you need to export it?

Reply:
It is being exported

Reply:
Yes, I know. I just don't see reason for this to be exported but it's ok to keep like this also

Reply:
I think it's nice to export it so we can test it later on